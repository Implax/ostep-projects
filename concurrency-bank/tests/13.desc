Correctness test: multiple negative transactions
