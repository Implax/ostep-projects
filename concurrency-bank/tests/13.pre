ln -sf "$(pwd)/tests/assets/negative_transactions.txt" Husband.txt && ln -sf "$(pwd)/tests/assets/no_transactions_wel.txt" Wife.txt
