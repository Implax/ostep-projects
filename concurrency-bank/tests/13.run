./bank 400 tests/assets/negative_transactions.txt tests/assets/no_transaction_wel.txt