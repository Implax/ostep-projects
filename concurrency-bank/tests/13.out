Opening balance: 400
Withdraw: -600, User: Husband, Transaction declined
Withdraw: -600, User: Husband, Transaction declined
Withdraw: -600, User: Husband, Transaction declined
Withdraw: -600, User: Husband, Transaction declined
Closing balance: 400
